benefits/static/css/styles.css

```diff
@@ -195,6 +199,14 @@ h2,
   line-height: var(--heading-line-height);
 }
 
+@media (max-width: 992px) {
+  /* H2: 20px, up to Small width */
+  .h2-sm {
+    font-size: var(--font-size-20px);
+    line-height: var(--heading-line-height);
+  }
+}
+
```

Comment:
The #1613 PR will build off of this. Fixing the Help page first with this PR will allow me to have a place to actually test #1613's responsive H2's rendering.